doc/api/os.md

```diff
@@ -452,6 +452,21 @@ On POSIX systems, the operating system release is determined by calling
 available, `GetVersionExW()` will be used. See
 <https://en.wikipedia.org/wiki/Uname#Examples> for more information.
 
+## `os.machine()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns {string}
+
+Returns the machine type as a string, such as `x86_64`.
```

Comment:
You should document the possible values, otherwise this is not as helpful as it could be.

Off the top of my head: arm, aarch64, mips, mips64, ppc64, ppc64le, s390, s390x, i386, i686, x86_64, and probably others

Both Linux and Windows can return either i386 or i686.

Reply:
Thanks. I have added this to doc.